EIPS/eip-5313.md

```diff
@@ -4,7 +4,8 @@ title: Light Contract Ownership
 description: An interface for identifying ownership of contracts
 author: William Entriken (@fulldecent)
 discussions-to: https://ethereum-magicians.org/t/eip-5313-light-contract-ownership/10052
-status: Draft
+status: Last Call
+last-call-deadline: 2022-08-05
```

Comment:
I would like to push this back a bit further (maybe to end of month). Draft -> Last Call is quite the jump!

Reply:
Updated this PR to move to Review. This can merge now. And then will make a separate one for Last Call. What can do later.